packages/devtools_app/test/test_infra/fixtures/memory_app/lib/main.dart

```diff
@@ -35,16 +35,16 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   final _garbage = <_MyGarbage>[];
   int _counter = 0;
-  _MyGarbage _gcableItem = _MyGarbage(0);
+  _MyGarbage _gcableItem = _MyGarbage(0, 'Should be gced, initial.');
```

Comment:
is this memory_app used in any tests? I did a search in our codebase and didn't see any tests that use this app

Reply:
For now I use it for manual testing. I plan to use it for integration testing in future.